man/en/sos-upload.1

```diff
@@ -0,0 +1,149 @@
+.TH UPLOAD 1 "December 2024"
+
+.SH NAME
+sos_upload \- Upload files like previously generated sos reports or logs to a policy specific location
+.SH SYNOPSIS
+.B sos upload FILE [options]
+          [--case-id id]\fR
+          [--upload-url url]\fR
+          [--upload-user user]\fR
+          [--upload-pass pass]\fR
+          [--upload-directory dir]\fR
+          [--upload-method]\fR
+          [--upload-no-ssl-verify]\fR
+          [--upload-protocol protocol]\fR
+          [--upload-target target]\fR
+          [--upload-s3-endpoint  endpoint]\fR
+          [--upload-s3-region region]\fR
+          [--upload-s3-bucket bucket]\fR
+          [--upload-s3-access-key key]\fR
+          [--upload-s3-secret-key key]\fR
+          [--upload-s3-object-prefix prefix]\fR
+
+.PP
+.SH DESCRIPTION
+upload is an sos subcommand to upload sos reports, logs, vmcores, or other files to a policy defined remote location, or a user defined one.
```

Comment:
Nitpick: put this to multiple lines likewise e.g. options? Or likewise https://github.com/sosreport/sos/blob/main/man/en/sos-report.1#L49 .

Reply:
Fixed, I think